test/gtest/cfg.cpp

```diff
@@ -147,11 +147,19 @@ End
   parseWast(wasm, moduleText);
 
   CFG cfg = CFG::fromFunction(wasm.getFunction("bar"));
-  FinitePowersetLattice lattice(wasm.getFunction("bar")->getNumLocals());
-  LivenessTransferFunction transferFunction;
+  size_t numLocals = wasm.getFunction("bar")->getNumLocals();
 
-  MonotoneCFGAnalyzer<FinitePowersetLattice, LivenessTransferFunction> analyzer(
-    lattice, transferFunction);
+  std::vector<Index> localIndices(numLocals);
+
+  for (Index i = 0; i < numLocals; ++i) {
+    localIndices[i] = i;
+  }
```

Comment:
Maybe we could add a constructor to handle this common case of the trivial `f(x)=x` mapping?

Reply:
If we use `FiniteIntPowersetLattice` now, we should no longer need handle this trivial case. Of course, I can still additionally implement this for `FinitePowersetLattice`.